### ISSUE TITLE
[BUG] Can't decorate files with multiple packages that have same path package name

### ISSUE THREAD

**Issue by mlund01** [DataDog/orchestrion#56]
When a file uses two packages that have different published names but the same package path name, the decorator errors out. An example error is `Error: Failed to scan: error scanning file model/customer.go: error decorating file model/customer.go: goast.DecoratorResolver found multiple packages using name v3: github.com/chargebee/chargebee-go/v3 and github.com/gocardless/gocardless-pro-go/v3`

**Version of orchestrion**
v0.4.1

**Describe what happened:**
Running orchestrion errored out when it found two packages with the same package path name.

**Describe what you expected:**
Orchestrion should ideally be smart enough to know what the underlying package name is. It is very common for third-party clients to version their packages with `/client/v1`, etc.. so these sorts of clashes are going to happen a bit without orchestrion knowing the published package name.

**Steps to reproduce the issue:**
create a file that uses the two packages in the workaround below and attempt to run orchestrion

**Workaround**
If you give the package an alias name  on import (even the one that's actually being used under the hood) it fixes the problem for orchestrion. For example,
```go
import (
	chargebee "github.com/chargebee/chargebee-go/v3"
	gocardless "github.com/gocardless/gocardless-pro-go/v3"
)
```

**Comment by RomainMuller** [comment]
We're working on a relatively drastic `orchstrion` refactor/reshape, and this already includes the necessary support for _correctly_ resolving those import paths instead of incorrectly processing both as `v3`.

We are trying to get a first iteration of the "new orchestion" available in the near future...

**Comment by RomainMuller** [comment]
We have now released `orchestrion@v0.7.1` which ought to address this issue, and has a new mode of operation. I'll close this issue now, but I encourage you to try it & let us know if you encounter any issues!
